index.html

```diff
@@ -20,11 +20,20 @@
 
       <div class="content">
 
+        <form action="information-about-book" method="POST">
+
```

Comment:
Be sure to add a `<label>` element or `placeholder` property to each `<input>` field